charts/tidb-lightning/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for tidb-lightning.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: pingcap/tidb-lightning:v3.0.1
+imagePullPolicy: IfNotPresent
+service:
+  type: NodePort
+
+dataSource:
+  local:
+    # nodeName: kind-worker3
+    hostPath: /data/export-20190820
+  remote:
+    rcloneImage: tynor88/rclone
```

Comment:
Here we use the latest `rclone` image because other versions don't support `-P` option which shows the copying progress.